example/example/tests/test_customer_views.py

```diff
@@ -366,3 +366,65 @@ def test_delete_customer_user_sends_email(self, admin, user, mock_storage):
         email = mail.outbox[0]
         assert len(mail.outbox) == 1
         assert user.email in email.to
+
+    def test_resend_invitation_new_customer_user(self, admin, user_data, mock_storage):
+
+        customer = CustomerFactory(logo=None)
+
+        _, key = create_auth_token(admin)
+        client = APIClient()
+        client.credentials(HTTP_AUTHORIZATION=f"Token {key}")
```

Comment:
These 3 lines are replicated in each test, I would look to factor them out into a helper function to save on this, but I'm not forcing the issue. There are many similarities between tests, with minor differences e.g. `GET` vs `POST` calls, urls etc. Again, I would look to try to factor this out, so the tests are as small as possible. I might take that on as a learning exercise, if you think it would be a good idea.

Reply:
That sounds fine to me.  Maybe a fixture that takes a user and returns a client w/ that user already authenticated?